node_modules/oae-email/lib/api.js

```diff
@@ -362,7 +362,9 @@ var sendEmail = module.exports.sendEmail = function(templateModule, templateId,
 
     // If the `from` header is not set, we generate an intelligent `from` header based on the tenant host
     var tenant = TenantsAPI.getTenant(toUser.tenant.alias);
-    var from = EmailConfig.getValue(tenant.alias, 'general', 'from') || util.format('"Apereo OAE" <noreply@%s>', tenant.host);
+    var fromName = EmailConfig.getValue(tenant.alias, 'general', 'fromName') || 'Apereo OAE';
```

Comment:
Note that this now defaults to `Apereo OAE`. However, it should default to the tenant displayname instead. @stuartf : Can you submit a follow-up PR for this?